content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -625,174 +474,72 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 
-### End-user authentication with per-path requirements
-
-End-user authentication can be enabled or disabled based on request path. This is useful if you want to
-disable authentication for some paths, for example, the path used for health check or status report.
-You can also specify different JWT requirements on different paths.
-
-{{< warning >}}
-The end-user authentication with per-path requirements is an experimental feature in Istio 1.1 and
-is **NOT** recommended for production use.
-{{< /warning >}}
-
-#### Disable End-user authentication for specific paths
+### Enforce valid token must be presented.
 
-Modify the `jwt-example` policy to disable End-user authentication for path `/user-agent`:
+To reject request without valid token, you will need to add an authorization rule to `deny` request without `requestPrincipal` as below. Request principal is available only when a valid JWT token is provided, so the rule is equivalent to deny requests without valid token.
 
 {{< text bash >}}
-$ cat <<EOF | kubectl apply -n foo -f -
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+kubectl apply -f - <<EOF
+apiVersion: "security.istio.io/v1beta1"
+kind: "AuthorizationPolicy"
 metadata:
-  name: "jwt-example"
+  name: "frontend-ingress"
+  namespace: istio-system
+  labels:
+    demo.istio.io: security
 spec:
-  targets:
-  - name: httpbin
-  origins:
-  - jwt:
-      issuer: "testing@secure.istio.io"
-      jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
-      trigger_rules:
-      - excluded_paths:
-        - exact: /user-agent
-  principalBinding: USE_ORIGIN
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
 EOF
 {{< /text >}}
 
-Confirm it's allowed to access the path `/user-agent` without JWT tokens:
-
-{{< text bash >}}
-$ curl $INGRESS_HOST/user-agent -s -o /dev/null -w "%{http_code}\n"
-200
-{{< /text >}}
-
-Confirm it's denied to access paths other than `/user-agent` without JWT tokens:
+Retry request without token, you should see it now fail with error code 403.
 
 {{< text bash >}}
 $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
-401
+403
 {{< /text >}}
 
-#### Enable End-user authentication for specific paths
+### Enforce valid token per-path
 
-Modify the `jwt-example` policy to enable End-user authentication only for path `/ip`:
+You can also refine the requirement per host, path, method etc with authorization. For example, you can change the authorization policy in the last section to only require JWT on `/headers`. After applying this authorization rule, request to `$INGRESS_HOST/headers` will fail with 403 code. But for all other path, e.g `$INGRESS_HOST/ip`, it should succeed. Please see other [tasks with authorization]() to learn more about other ways to refine access control with authoriation policy.
```

Comment:
I removed the link to the other authorization tasks since it was empty.

```suggestion
To refine authorization with a token requirement per host, path, or method, change the authorization policy to only require JWT on `/headers`. When this authorization rule takes effect, requests to `$INGRESS_HOST/headers` fail with the error code `403`. Requests to all other paths succeed, for example `$INGRESS_HOST/ip`. 
```

Reply:
done